src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolver.Session.cs

```diff
@@ -537,7 +537,7 @@ private async Task DebugVerifyNoErrorsAsync(MutableConflictResolution conflictRe
                             foreach (var originalReference in conflictAnnotation.RenameDeclarationLocationReferences.Where(loc => loc.IsSourceLocation))
                             {
                                 var adjustedStartPosition = conflictResolution.GetAdjustedTokenStartingPosition(originalReference.TextSpan.Start, originalReference.DocumentId);
-                                if (newLocations.Any(loc => loc.SourceSpan.Start == adjustedStartPosition))
+                                if (newLocations.Any(loc => loc!.SourceSpan.Start == adjustedStartPosition))
```

Comment:
This is a down side of lacking support of `WhereNotNull`. Does this require a comment?

Reply:
I didn't mention you defined a `WhereNotNull`. Totally fine.